tempesta_fw/cache.c

```diff
@@ -1399,7 +1407,9 @@ tfw_cache_cfg_method(TfwCfgSpec *cs, TfwCfgEntry *ce)
 	unsigned int i, method;
 	const char *val;
 
-	BUILD_BUG_ON(sizeof(cache_cfg.methods) * 8 < _TFW_HTTP_METH_COUNT);
+	BUILD_BUG_ON(sizeof(cache_cfg.methods) * BITS_PER_BYTE
+		     < _TFW_HTTP_METH_COUNT);
```

Comment:
My understanding of `BUILD_BUG_ON()` is that it's evaluated at compilation time, so using it once per a condition is sufficient.